backup/backup.go

```diff
@@ -188,6 +188,181 @@ func (c *BackupCommand) BackupInfo(cliConnection plugin.CliConnection, backupId
 
 }
 
+func (c *BackupCommand) ListBackupsByInstanceGuid(cliConnection plugin.CliConnection, serviceInstanceGuid string) {
+	fmt.Println("Getting the list of  backups in the org", AddColor(helper.GetOrgName(helper.ReadConfigJsonFile()), cyan), "/ space", AddColor(helper.GetSpaceName(helper.ReadConfigJsonFile()), cyan), "/ service instance GUID", AddColor(serviceInstanceGuid, cyan), "...")
+
+	if helper.GetAccessToken(helper.ReadConfigJsonFile()) == "" {
+		errors.NoAccessTokenError("Access Token")
+	}
+
+	client := GetHttpClient()
+
+	var userSpaceGuid string = helper.GetSpaceGUID(helper.ReadConfigJsonFile())
+
+	var guid string = serviceInstanceGuid
+	var ServiceName string = guidTranslator.FindInstanceName(cliConnection, guid, nil)
+
+	if ServiceName == "" {
+		errors.IncorrectGuid(guid)
+		return
+	}
+
+	var apiEndpoint string = helper.GetApiEndpoint(helper.ReadConfigJsonFile())
+	var broker string = GetBrokerName()
+	var extUrl string = GetExtUrl()
+
+	apiEndpoint = strings.Replace(apiEndpoint, "api", broker, 1)
+
+	req, err := http.NewRequest("GET", apiEndpoint+extUrl+"/backups"+"?space_guid="+userSpaceGuid, nil)
```

Comment:
The idea is to list backups even if instance is deleted. Hence you need to use `/backups?space_guid=&instance_id=` api here as per api doc (broker_v1.0.html)[https://cloudfoundry-incubator.github.io/service-fabrik-broker/api/broker_v1.0.html].

Reply:
I think this api endpoint has been introduced recently.
Thus, we can now also modify the function, `ListBackupsByInstanceName` to use the same api endpoint and hence, avoid the filtering (it was earlier getting all results and later filtering out based on the instance name).
The logic now could be:
When user provides an instance_name, extract the guid out of it and then hit the api endpoint with the guid. 
This will improve the computational efficiency of the command by multiples!